src/ImageSharp/Formats/Webp/Lossless/HuffmanUtils.cs

```diff
@@ -202,9 +202,14 @@ public static void GenerateOptimalTree(HuffmanTree[] tree, uint[] histogram, int
                 }
 
                 // Build the Huffman tree.
+#if NET5_0_OR_GREATER
+                Span<HuffmanTree> treeSlice = tree.AsSpan().Slice(0, treeSize);
```

Comment:
```suggestion
                Span<HuffmanTree> treeSlice = tree.AsSpan(0, treeSize);
```
Also applicable on line 209

Reply:
changed with e97c364b